frontend/src/component/HomePage.tsx

```diff
@@ -70,16 +73,37 @@ export default function HomePage() {
         navigate("/topic/add")
     }
 
+    const onNodeClickDelete: NodeMouseHandler = (event, node) => {
+        if (deleteMode && node.data.label !== "HOME") {
+            deleteSingleTopic(node.id);
+        }
+    }
+
+    function displayHint() {
+        if (deleteMode) {
+            return (<p className={"hint-text"}>CLICK TOPIC TO DELETE</p>)
+        }
+    }
+
     return (
         <div style={{width: '100vw', height: '100vh'}}>
-            <button className="addButton" onClick={handleSubmitAdd}></button>
+            {displayHint()}
+            <button className="addButton" onClick={handleSubmitAdd}>
+                <AddIcon sx={{fontSize: 40, color: "white"}}/>
+            </button>
+            <button className={deleteMode ? "delete-active" : "delete"} onClick={() => {
+                !deleteMode ? setDeleteMode(true) : setDeleteMode(false);
```

Comment:
I think the React-proposed way for this is:
```suggestion
                setDeleteMode(oldValue => !oldValue);
```